src/components/Composer.vue

```diff
@@ -619,6 +723,7 @@ export default {
 				messageId: this.replyTo ? this.replyTo.databaseId : undefined,
 				isHtml: !this.editorPlainText,
 				requestMdn: this.requestMdn,
+				sendAt: this.sendAt,
```

Comment:
the converstion from js timestamp to unix timestamp is missing


add the /1000 here

![Bildschirmfoto vom 2022-04-08 17-48-14](https://user-images.githubusercontent.com/1374172/162477394-eeaa218c-46ec-4e98-9867-d928366242ef.png)